flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/sql/JoinTest.scala

```diff
@@ -55,12 +55,13 @@ class JoinTest extends TableTestBase {
           unaryNode(
             "DataStreamCalc",
             streamTableNode(1),
-            term("select", "a", "b", "proctime")
+            term("select", "a", "b", "-(proctime, 3600000) AS -",
```

Comment:
In the original test, the calculation of **proctime** is located when doing **Join**. While currently, it's pushed down to the **DataStremaCalc**. It seems that this is not correct?